services/submission.js

```diff
@@ -1117,6 +1132,41 @@ const isSubmissionPermitted = (aSubmission, userInfo) => {
     throw new Error(ERROR.INVALID_STATS_SUBMISSION_PERMISSION);
 }
 
+class ValidationRecord {
+    // submissionID: string
+    // type: array
+    // scope: array
+    // started: Date
+    // status: string
+    constructor(submissionID, type, scope, status) {
+        this._id = v4();
+        this.submissionID = submissionID;
+        this.type = type;
+        this.scope = scope;
+        this.started = getCurrentTime();
+        this.status = status;
+    }
+    static createValidation(submissionID, validationType, validationScope, status) {
+        return new ValidationRecord(submissionID, validationType, validationScope, status);
+    }
+}
+
+
+class DataValidation {
+    // validationType: string
+    // validationScope: string
+    // validationStarted: Date
+    constructor(validationType, validationScope, validationStarted) {
+        this.validationStarted = validationStarted ? validationStarted : getCurrentTime();
+        this.validationEnded = null;
+        this.validationType = validationType?.map(type => type.toLowerCase());
+        this.validationScope = validationScope?.toLowerCase();
+    }
+    static createDataValidation(validationType, validationScope, validationStarted) {
+        return new DataValidation(validationType, validationScope, validationStarted);
+    }
+}
```

Comment:
Please update the "submission" type in the graphql schema to add these new fields so that they can be queried

Reply:
updated